### PR TITLE
feature(OH2-283): admin exams list

### PR DESCRIPTION
**Pull request by gasp**
See See [OH2-283](https://openhospital.atlassian.net/browse/OH2-283)

This is the first chunk of the admin/exams page, other chunks will follow such as CRUD editing and filters

List exams

[OH2-283]: https://openhospital.atlassian.net/browse/OH2-283?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/accessories/table/Table.tsx [hunk 1]
```diff
@@ -233,11 +233,13 @@ const Table: FunctionComponent<IProps> = ({
...
-          {expanded ? t("common.collapse_all") : t("common.expand_all")}
-        </Button>
-      </div>
+      {isCollapsabile ?? (
+        <div className="header">
+          <Button type="button" onClick={handleExpand}>
+            {expanded ? t("common.collapse_all") : t("common.expand_all")}
+          </Button>
+        </div>
+      )}
```
Comment (SteveGT96) [inline src/components/accessories/table/Table.tsx]:
```suggestion
        <div className="header">
        {!!isCollapsabile && (
          <Button type="button" onClick={handleExpand}>
            {expanded ? t("common.collapse_all") : t("common.expand_all")}
          </Button>
          )}
        </div>
```

**Review by SteveGT96 — CHANGES_REQUESTED** [review]

File: src/components/accessories/table/Table.tsx (hunk 1 above)
Comment (SteveGT96) [reply src/components/accessories/table/Table.tsx]: Hello @gasp ! Great Job. I think with this suggestion, we could move on...

Comment (gasp) [reply src/components/accessories/table/Table.tsx]: Thanks Steve for the review and the suggestion !

**Review by SteveGT96 — APPROVED** [review]

**Review by mwithi — APPROVED** [review]
Approved by @SteveGT96

**Pull request merged**